src/_docs/extensions/python.md

```diff
@@ -183,3 +183,88 @@ Ids are guaranteed to be unique. This means that if several of those paths conta
 Python exceptions thrown at the core application are not handled well and may introduce a lot of weird errors. Ensure that you catch _all_ exceptions in the Python code.
 
 The Python interpreter shuts down when the Python extension is unloaded. After this, enabling the extension will restart the interpreter. Some modules cannot be re-initialized safely and may cause segfaults after the interpreter has been restarted (numpy!). The issue is that Python itself cannot completely unload extension modules and there are several caveats with regard to interpreter restarting. In short, not all memory may be freed, either due to Python reference cycles or user-created global data. All the details can be found in the CPython documentation.
+
+# Example
+
+In this example we'll create a python script to generate random UUIDs. The `initialize` method, `finalize` method, and `__dependencies__` checking can be removed if you do not need them. For more plugin examples, see the [albert python plugins repository](https://github.com/albertlauncher/python).
+
+1. Create the folder `~/.local/share/albert/org.albert.extension.python/modules/uuid`
+2. Create a file `~/.local/share/albert/org.albert.extension.python/modules/uuid/__init__.py`
+3. In that file, place the UUID script from below
+4. Open Albert settings and navigate to Extensions > Python
+5. If the "UUID" option is not listed, fully quit and re-open Albert
+6. Check the checkbox next to UUID, the light should turn green
+7. Use the extension by triggering albert and typing 'uuid'
+
+If the light turns red, there was an error loading your extension - hover over the name to view the details.
+
+It can be useful for debugging to run Albert from a terminal so that you can see the output from your script.
+
+Though reloading Albert is may be necessary to recognize that a python script is added, it is not required to detect changes to the script.
```

Comment:
here typo

Reply:
Ah; I accidentally left the word “is” in; got it 👍